### PR TITLE
Added support for COPYABLE_ANNOTATIONS config

### PR DESCRIPTION
**Pull request by sluongng**
Added support for Lombok v1.18.4 config `lombok.copyableAnnotations`

### CODE REVIEW

**Comment by mplushnikov** [comment]
@sluongng Thank you for your work, but it doesn't add any new functionality. It only adds definition of the configuration key. This definition is not used anywhere.

**Comment by sluongng** [comment]
> @sluongng Thank you for your work, but it doesn't add any new functionality. It only adds definition of the configuration key. This definition is not used anywhere.

@mplushnikov its used for intellij autocompletion when you type inside the lombok.config file

perhaps in a far distant future you can make a custom menu to generate the file according to these definition, but lets add this for now.

A reason why this config is important is because it enable retention of annotation such as @Qualifier in Spring for autowiring via constructor(which is generated by lombok).

Please do consider merging this as a first step and we can improve it iteratively after.

**Comment by mplushnikov** [comment]
@sluongng For autocompletion inside of lombok.config file you have to change LombokConfigCompletionContributor implementation too.

**Comment by sluongng** [comment]
Hmm sorry I am not that experienced with IntelliJ Plugin development.
Added the change to mentioned file

File: src/main/java/de/plushnikov/intellij/plugin/language/LombokConfigCompletionContributor.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public LombokConfigCompletionContributor() {
 
     final Collection<String> flagUsageOptions = new HashSet<>(Arrays.asList(
       "lombok.accessors.flagUsage", "lombok.allArgsConstructor.flagUsage", "lombok.anyConstructor.flagUsage",
-      "lombok.builder.flagUsage", "lombok.cleanup.flagUsage", "lombok.data.flagUsage", "lombok.delegate.flagUsage",
+      "lombok.builder.flagUsage", "lombok.cleanup.flagUsage", "lombok.copyableAnnotations", "lombok.data.flagUsage", "lombok.delegate.flagUsage",
```
Comment (mplushnikov) [inline src/main/java/de/plushnikov/intellij/plugin/language/LombokConfigCompletionContributor.java]: Why did you add it here, in the middle of only flagUsage options? Why do you think this is a good idea?

Maybe you can add it to "final Collection<String> otherOptions"?  And use a reference to your created key in from ConfigKey. (ConfigKey.COPYABLE_ANNOTATIONS..getConfigKey()). I think this would make more sense:)
